packages/legend-query-builder/src/components/__tests__/QueryBuilderResultPanel.test.ts

```diff
@@ -133,6 +133,51 @@ describe(integrationTest('Query builder result state'), () => {
         queryBuilderState.resultState.setExecutionResult(executionResult);
       });
 
+      // Test filter out/by after sorting column
```

Comment:
lets please enhance the execution result loaded here to be a bit more complex. With links, null values, number as strings etc. 

Also what erorr do you get with context menu?